frontend/src/assets/css/scaffolding.scss

```diff
@@ -30,6 +30,7 @@ select {
   display: flex;
   flex-grow: 1;
   flex-direction: column;
+  height: 100vh;
```

Comment:
Here height will be with fixed value. It seems not to be OK? @andanfe

Reply:
Yep, and the main purpose of fixing root element height, in this case, is to prevent it's growth over the screens height and provide scrolling only for its contents, as it was requested in ticket 🤔
But in this case `flex-grow: 1` may be removed

Reply:
agreed. use `flex-grow` instead